x/emissions/migrations/v4/migrate.go

```diff
@@ -0,0 +1,67 @@
+package v4
+
+import (
+	errorsmod "cosmossdk.io/errors"
+	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/zeta-chain/node/x/emissions/types"
+)
+
+type EmissionsKeeper interface {
+	SetParams(ctx sdk.Context, params types.Params) error
+	GetCodec() codec.BinaryCodec
+	GetStoreKey() storetypes.StoreKey
+}
+
+// MigrateStore migrates the store from v3 to v4
+// The v3 params are copied to the v4 params, and the v4 params are set in the store
+// v4 params removes unused parameters from v3; these values are discarded.
+// v4 introduces a new parameter, BlockRewardAmount, which is set to the default value
+func MigrateStore(
+	ctx sdk.Context,
+	emissionsKeeper EmissionsKeeper,
+) error {
+	v3Params, found := GetParamsLegacy(ctx, emissionsKeeper.GetStoreKey(), emissionsKeeper.GetCodec())
+	if !found {
+		return errorsmod.Wrap(types.ErrMigrationFailed, "failed to get legacy params")
+	}
+
+	v4Params := types.NewParams()
+	if v3Params.ValidatorEmissionPercentage != "" {
+		v4Params.ValidatorEmissionPercentage = v3Params.ValidatorEmissionPercentage
+	}
+	if v3Params.ObserverEmissionPercentage != "" {
+		v4Params.ObserverEmissionPercentage = v3Params.ObserverEmissionPercentage
+	}
+	if v3Params.TssSignerEmissionPercentage != "" {
+		v4Params.TssSignerEmissionPercentage = v3Params.TssSignerEmissionPercentage
+	}
+	v4Params.ObserverSlashAmount = v3Params.ObserverSlashAmount
```

Comment:
I don't think we need to validate , this because `SetParams` validates before setting these params. 
I think the empty validations are a bit unnecessary as well , but we can just keep them for now